app/graphql/queries/survey_query.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Queries
+  class SurveyQuery < BaseQuery
+    type [Types::SurveyType], null: false
+    description 'Survey list'
+
+    def resolve
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/46106ccd13ad8d3333b75e32a92764a807848798/app/graphql/queries/survey_query.rb#L8"></span>Doesn’t depend on instance state (maybe move it to another class?)</td>
    </tr>
  </tbody>
</table>